### PR TITLE
Disable KBFS install on sierra

### PR DESCRIPTION
**Pull request by gabriel**
screen shot 2016-09-20 at 6 26 53 pm

### CODE REVIEW

File: go/install/install_osx.go [hunk 1]
```diff
@@ -417,13 +417,26 @@ func Install(context Context, binPath string, components []string, force bool, l
...
 	return newInstallResult(componentResults)
 }
 
+func isKBFSCompatible(log Log) bool {
+	osver, err := OSVersion()
+	if err != nil {
+		log.Warning("Error getting OS version: %s", err)
+		return false
+	}
+	return !(osver.Major == 10 && osver.Minor == 12)
```
Comment (maxtaco) [inline go/install/install_osx.go]: Maybe: `!(osver.Major > 10 || (osver.Major == 10 && osver.Minor >= 12))`?

Comment (gabriel) [inline go/install/install_osx.go]: I think it's ok to target 10.12 exactly... Fuse has to be whitelisted for each OS version anyway so...

**Comment by gabriel** [comment]
If kbfs isn't installed then they won't have a /keybase and so fs related wouldn't really do anything?

**Comment by maxtaco** [comment]
ok, 👍  with sadface though!

**Comment by cjb** [comment]
Could you be a bit more specific about the user visible behavior? Do they see favorites in the widget? What happens if they click on one, nothing?

**Comment by cjb** [comment]
(Not holding up the merge, just curious.)

**Comment by gabriel** [comment]
Yeah they see favorites... when they click on a favorite, it won't open Finder cause the directory isn't there.

**Comment by cjb** [comment]
@gabriel How does this help users who already had Keybase and KBFS installed, then upgraded to Sierra?  (Isn't that approximately all of our users?)

**Comment by gabriel** [comment]
KBFS isn't running until the app starts/installs it (the launchd service is created on app start)

**Comment by cjb** [comment]
@gabriel Yes, but they had a launchd service that already existed on El Capitan.  Are you saying that the Sierra installs deletes all launchd services?

**Comment by gabriel** [comment]
We changed it recently that stopping the app, removes the launchd service (plist) (instead of just stopping it).

**Comment by gabriel** [comment]
Which is currently in public release (1.0.17-20160913233827+18b5a25)

**Comment by cjb** [comment]
Ah, that makes sense, but KBFS was running after my upgrade, which doesn't make sense.  Filed a JIRA.

**Comment by gabriel** [comment]
https://github.com/keybase/client/pull/4370

**Comment by gabriel** [comment]
This here ensures that we don't RunAtLoad our plists anymore: https://github.com/keybase/client/pull/4374

**Pull request merged**